deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Configured as part of the DigitalOcean 1-Click Image build process
+
+myip=$(hostname -I | awk '{print$1}')
+cat <<EOF
+********************************************************************************
+
+Welcome to VictoriaMetrics server!
+To keep this server secure, the UFW firewall is enabled.
+All ports are BLOCKED except 22 (SSH), 80 (HTTP), and 443 (HTTPS), 8428 (VictoriaMetrics HTTP), 8089 (VictoriaMetrics Influx),
+4242 (VictoriaMetrics OpenTSDB), 2003 (VictoriaMetrics Graphite)
+
+In a web browser, you can view:
+ * The VictoriaMetrics Quickstart guide: https://kutt.it/1click-quickstart
+
+On the server:
+  * The default VictoriaMetrics root is located at /var/lib/victoria-metrics-data
+  * VictoriaMetrics is running on ports: 8428, 8089, 4242, 2003 and they are bound to the local interface.
+
+********************************************************************************
+  # This image includes version v1.81.2 of VictoriaMetrics. 
```

Comment:
Should version here be templated from build variable?

Reply:
yep, fixed